IBPSA/Fluid/Sources/PropertySource_h.mo

```diff
@@ -0,0 +1,62 @@
+within IBPSA.Fluid.Sources;
+model PropertySource_h
+  "Model for overriding fluid properties that flow through the component, using specific enthalpy input"
+  parameter Boolean use_h_in= false
+    "Set to true to get the specific enthalpy from the input connector"
+    annotation(Evaluate=true, Dialog(group="Inputs"));
+
+  extends IBPSA.Fluid.Sources.BaseClasses.PartialPropertySource;
+
+  Modelica.Blocks.Interfaces.RealInput h_in if use_h_in
+    "Prescribed value for specific enthalpy" annotation (Placement(
+        transformation(
+        extent={{-20,-20},{20,20}},
+        rotation=270,
+        origin={-40,120})));
+equation
+  connect(h_internal_a, h_in);
+  connect(h_internal_b, h_in);
+  if not (use_h_in) then
+    connect(h_internal_a,h_in_b);
+    connect(h_internal_b,h_in_a);
+  end if;
+annotation (defaultComponentName="proSou",
+        Icon(coordinateSystem(preserveAspectRatio=true,  extent={{-100,
+            -100},{100,100}}), graphics={
+        Text(
+          visible=use_h_in,
+          extent={{-90,98},{12,58}},
+          lineColor={0,0,0},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          textString="h")}),
+    Documentation(info="<html>
+<p>
+Model that changes the properties, 
+but not the mass flow rate,
+of the fluid that passes through it.
+</p>
+<h4>Typical use and important parameters</h4>
+<p>
+The fluid properties <code>h</code>, <code>Xi</code> and <code>C</code>
+are only modified when the corresponding boolean parameter
+<code>use_h_in</code>, <code>use_Xi_in</code> or <code>use_C_in</code>
+is enabled.
```

Comment:
you mean "...is set to <code>true</code>", as these parameters are always enabled.